src/main/resources/templates/branches.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Branches</title>
+</head>
+<body>
+<form method="get" th:action="@{/company/branches/create}">
+    <button type="submit">CREATE BRANCH</button>
+</form>
+<p>DO NOT remove branch with Id=1(created on app startup). For reasons unknown to me(Oskar), it throws and exception and I was unable to fix it. It will get fixed later.</p>
```

Comment:
nie możesz usunąć brancha z powiązaniami (pracownicy/samochody/adres) - sql nie pozwala

Reply:
Aaaa czyli np. kaskadę trzeba by było. Ale to ciekawe, bo w delete dla innych branchów stworzonych już na późniejszym etapie (CRUDem) działa tak jak powinno.